packages/functional-tests/tests/settings/changePassword.spec.ts

```diff
@@ -3,105 +3,147 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { test, expect } from '../../lib/fixtures/standard';
-let newPassword;
 
 test.describe('severity-1 #smoke', () => {
   test.describe('change password tests', () => {
-    test.beforeEach(async ({ pages: { settings }, credentials }) => {
-      newPassword = credentials.password + '@@2';
-      await settings.goto();
-      await settings.password.clickChange();
-    });
-
     test('change password with an incorrect old password', async ({
-      pages: { changePassword },
+      credentials,
+      pages: { settings, changePassword },
     }) => {
       test.slow();
+
+      await settings.goto();
+
       // Enter incorrect old password and verify the tooltip error
+      await settings.password.changeButton.click();
+      const newPassword = credentials.password + '@@2';
       await changePassword.fillOutChangePassword(
         'Incorrect Password',
-        newPassword
-      );
-      await changePassword.clickSignIn();
-      expect(await changePassword.changePasswordTooltip()).toContain(
-        'Incorrect password'
+        newPassword,
+        true
       );
+
+      await expect(changePassword.tooltip).toHaveText('Incorrect password');
     });
 
     test('change password with a correct password', async ({
-      pages: { settings, changePassword, login },
       credentials,
+      pages: { settings, changePassword, login },
     }) => {
       test.slow();
+
+      await settings.goto();
+
       // Enter the correct old password and verify that change password is successful
+      await settings.password.changeButton.click();
+      const newPassword = credentials.password + '@@2';
       await changePassword.fillOutChangePassword(
         credentials.password,
-        newPassword
+        newPassword,
+        true
       );
-      await changePassword.submit();
+      credentials.password = newPassword;
+
+      await expect(settings.settingsHeading).toBeVisible();
+      await expect(settings.alertBar).toHaveText('Password updated');
 
       // Sign out and login with new password
       await settings.signOut();
-      credentials.password = newPassword;
       await login.setEmail(credentials.email);
```

Comment:
Can change to `login.fillOutEmailFirstSignIn(credentials.email, credentials.password)`;

Reply:
see https://github.com/mozilla/fxa/pull/16671/commits/515929df9273ebd1f3b5c00ff640e7b6d6c1586a